Bottom: 25d785e7b89cebfa1d9dec62e1345323d956fa2c
Top:    e62cc5b2ff26ab9f84d6019c39f2277cbb28dcd2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:19 +0000

Disable process limits


---

diff --git a/src/polygon.c b/src/polygon.c
index 420772b..fb40faf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -132,6 +132,7 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
+return;
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
